pkg/grizzly/json.go

```diff
@@ -34,7 +34,7 @@ func (parser *JSONParser) Parse(file string, options ParserOptions) (Resources,
 		return nil, err
 	}
 
-	resources, err := parseAny(parser.registry, m, options.DefaultResourceKind, options.DefaultFolderUID)
+	resources, err := parseAny(parser.registry, m, options.DefaultResourceKind, options.DefaultFolderUID, "json", file, true)
```

Comment:
The signature of this function feels a bit overwhelming. From here, it's also not obvious what `true` means here.

Reply:
yah, time for a struct.

Reply:
Actually, with a `Source` struct, this gets much simpler.